### PR TITLE
Structure for fundamentals and front end resources

### PR DESCRIPTION
**Pull request by P4NK4J**
Changes include:

1. Crash course for dynamic landing page
2. Structure for the basics of getting started into programming and technologies

### CODE REVIEW

File: getting started/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+
+## Online resources
+#### Try not to take shortcut
+1. https://www.youtube.com/watch?v=3xq96I9IUXw
+
+
+
+#### Books
```
Comment (Satendra-SR) [inline getting started/README.md:13]:
```suggestion
## Books
```

File: getting started/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Getting Started
+
+This document covers the fundamental approach to tackle any problem arriving at the initial phase of the journey.
+
+
+
```
Comment (Satendra-SR) [inline getting started/README.md:6]:
```suggestion
```

**Review by Satendra-SR — APPROVED** [review]
Looks good. Added a few suggestions, have a look before merging.

File: getting started/README.md (hunk 1 above)
Comment (P4NK4J) [reply getting started/README.md]: Made changes in the latest commit. Thanks

File: getting started/README.md (hunk 2 above)
Comment (P4NK4J) [reply getting started/README.md]: Made changes before the merge

**Pull request merged**